README.md

```diff
@@ -123,8 +123,9 @@ Template variables:
 - `jira_template_file` - Jira issue template path, available only in Jira
   template prerequisites
 
-There are also `days` and `weeks` filters for creating Python `timedelta`
-objects for date manipulation. For example: `today + 1|days`
+There are also `days`, `day`, `weeks` and `week` filters for creating Python
+`timedelta` objects for date manipulation. For example: `today + 1|day` or
+`today + 1|days`, `start_date +2|weeks`.
```

Comment:
`1|days` is somewhat confusing. It would be also better to add note that the singular and plural versions are aliases and can take any number.

```suggestion
`today + 2|days`, `start_date +2|weeks`. Note: Singular and plural versions
work the same and take any integer as an argument. So both `2|day` and `1|days`
would work.
```